code/modules/mob/living/simple_animal/abnormality/teth/mhz.dm

```diff
@@ -0,0 +1,79 @@
+/mob/living/simple_animal/hostile/abnormality/mhz
+	name = "1.76 MHz"
+	desc = "You can't see anything but static."
+	icon = 'ModularTegustation/Teguicons/96x96.dmi'
+	icon_state = "mhz"
+	icon_living = "mhz"
+	pixel_x = -32
+	base_pixel_x = -32
+	pixel_y = -32
+	base_pixel_y = -32
+	maxHealth = 400
+	health = 400
+	start_qliphoth = 4
+	threat_level = TETH_LEVEL
+	work_chances = list(
+		ABNORMALITY_WORK_INSTINCT = 40,
+		ABNORMALITY_WORK_INSIGHT = list(40, 30, 20, 20, 20),
+		ABNORMALITY_WORK_ATTACHMENT = list(20, 10, 0, 0, 0),
+		ABNORMALITY_WORK_REPRESSION = list(55, 55, 60, 60, 60)
+		)
+	work_damage_amount = 5
+	work_damage_type = WHITE_DAMAGE
+
+	ego_list = list(
+		/datum/ego_datum/weapon/noise,
+		/datum/ego_datum/armor/noise
+		)
+
+/mob/living/simple_animal/hostile/abnormality/mhz/work_chance(mob/living/carbon/human/user, chance)
+	if(get_attribute_level(user, FORTITUDE_ATTRIBUTE) < 40)
+		var/newchance = chance*1.25
+		return newchance
+	return chance
+
+/mob/living/simple_animal/hostile/abnormality/mhz/success_effect(mob/living/carbon/human/user, work_type, pe)
+	datum_reference.qliphoth_change(1)
+	if(pe >= datum_reference.max_boxes)
+		datum_reference.qliphoth_change(1)
+
+/mob/living/simple_animal/hostile/abnormality/mhz/neutral_effect(mob/living/carbon/human/user, work_type, pe)
+	if(prob(40))
+		datum_reference.qliphoth_change(-1)
+	return
+
+/mob/living/simple_animal/hostile/abnormality/mhz/failure_effect(mob/living/carbon/human/user, work_type, pe)
+	if(prob(80))
+		datum_reference.qliphoth_change(-1)
+	return
+
+/mob/living/simple_animal/hostile/abnormality/mhz/work_complete(mob/living/carbon/human/user, work_type, pe, work_time)
+	if(user.sanity_lost)
+		datum_reference.qliphoth_change(-1)
+	return ..()
+
+/mob/living/simple_animal/hostile/abnormality/mhz/zero_qliphoth(mob/living/carbon/human/user)
+	SSweather.run_weather(/datum/weather/mhz)
```

Comment:
I honestly think using weather for everything isn't a good idea, they will override each other.

Reply:
I don't exactly think it's a _bad_ thing that you can only have a single crippling, station affecting abnormality effect at the same time, having both
Reduce defenses by 33%
And
Deal white damage steadily 
Would honestly be probably too much. We would just have to make sure that there's a priority station in place, to make it so that you can't cheese out more damaging effects, or run a callback after weather ends to run another one.